Brocfile.js

```diff
@@ -17,4 +17,15 @@ var app = new EmberApp();
 // please specify an object with the list of modules as keys
 // along with the exports of each module as its value.
 
+// Foundation
+app.import('bower_components/foundation/js/foundation.min.js');
+
+// FontAwesome
+app.import('bower_components/fontawesome/fonts/FontAwesome.otf', {destDir: 'assets/fontawesome/fonts'});
+app.import('bower_components/fontawesome/fonts/fontawesome-webfont.eot', {destDir: 'assets/fontawesome/fonts'});
+app.import('bower_components/fontawesome/fonts/fontawesome-webfont.svg', {destDir: 'assets/fontawesome/fonts'});
+app.import('bower_components/fontawesome/fonts/fontawesome-webfont.ttf', {destDir: 'assets/fontawesome/fonts'});
+app.import('bower_components/fontawesome/fonts/fontawesome-webfont.woff', {destDir: 'assets/fontawesome/fonts'});
+app.import('bower_components/fontawesome/fonts/fontawesome-webfont.woff2', {destDir: 'assets/fontawesome/fonts'});
+
```

Comment:
Wouldn't it be better to use [broccoli-static-compiler](http://www.ember-cli.com/#using-broccoli-static-compiler) to collect all the FontAwesome assets?